dist/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
-  "version": "3.0.0.42",
+  "version": "3.0.0",
```

Comment:
why remove the build # from here?

Reply:
Oh, I guess it was removed during one of the merges and I overlooked that. Will return it back

Reply:
thanks! feel free to merge afterwards